IntegrationWithPharmacies/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using HealthClinic.CL.DbContextModel;
+using HealthClinic.CL.Dtos;
+using HealthClinic.CL.Repositories;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using HealthClinic.CL.Model.Patient;
+using PatientWebApplication.Controllers;
+using HealthClinic.CL.Adapters;
+
+namespace IntegrationWithPharmacies
+{
+    public class Program
+    {
+        
+
+
+    public static void Main(string[] args)
+        {
+
+
+
+        }
```

Comment:
It is necessary to delete the main function because there is no implementation in it.